File: src/API/src/Modules/Clients/Clients.Application/DTOs/CreateClientContactRequest.cs
namespace Clients.Application.DTOs
{
    public class CreateClientContactRequest
    {
        public Guid ClientId { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string FullName { get; set; } = string.Empty;
        public string? Designation { get; set; }
        public string? Department { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public string? Mobile { get; set; }
        public bool IsPrimary { get; set; }
        public string? Notes { get; set; }
    }
}

File: src/ClientApp/src/app/pages/clients/dtos/create-client-contact.request.ts
export interface CreateClientContactRequest {
    clientId: string;
    firstName: string;
    lastName: string;
    designation?: string | null;
    department?: string | null;
    email?: string | null;
    phone?: string | null;
    mobile?: string | null;
    isPrimary: boolean;
    notes?: string | null;
}
